qcschema/dev/properties/cc_properties.py

```diff
@@ -0,0 +1,45 @@
+"""
+The complete list of Coupled-Cluster properties.
+"""
+
+cc_properties = {}
+
+cc_properties["ccsd_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The unscaled portion of the CCSD correlation energy from same-spin or triplet doubles correlations."
```

Comment:
well, I was going to criticize "unscaled portion" as vague, but that's what I have in the psi4 glossary, so my problem. :-)

maybe "The portion of CCSD doubles correlation energy from same-spin (i.e., triplet) correlations, without any user scaling."

Reply:
I'm happy with that change. Should I also update the opposite spin correlation energy description in the same manner? And should `mp_properties` descriptions be updated in the same way?

Reply:
Yes on both. I think it's a little clearer, and editing `description` is harmless.

Reply:
Sounds good, I have updated the descriptions.